gym/core.py

```diff
@@ -300,9 +295,6 @@ def step(self, action):
         action = self.action(action)
         return self.env.step(action)
 
-    def reset(self, **kwargs):
```

Comment:
wouldn't this cause warnings from default reset method of Wrapper class to show up?